proto/viam/robot/v1/robot.proto

```diff
@@ -360,3 +367,32 @@ message RestartModuleResponse {}
 message ShutdownRequest {}
 
 message ShutdownResponse {}
+
+message GetMachineStatusRequest {}
+
+message GetMachineStatusResponse {
+  repeated ResourceStatus resources = 1;
+}
+
+message ResourceStatus {
```

Comment:
note that error is currently omitted - it will be added once we implement the "unhealthy" state